protocol/chainlib/chainproxy/rpcclient/errors.go

```diff
@@ -33,6 +36,14 @@ func (err HTTPError) Error() string {
 	return fmt.Sprintf("%v: %s", err.Status, err.Body)
 }
 
+type InvalidStatusCodeError struct {
```

Comment:
Thats not what I meant I meant the cosmossdk error package see any other errors.go file (best example lavasession/errors.go)